sdk/src/gateway/types.ts

```diff
@@ -72,6 +72,18 @@ export interface Token {
 
 export interface EnrichedToken extends Token {
     tvl: number;
+    apys: {
+        id: 'base' | 'op';
+        name: string;
+        logo: string;
+        apy: number;
+    }[];
+    incentives: {
+        id: 'spice' | 'solvXP' | 'babylonPoints' | 'avalonPoints' | 'vedaPoints' | 'segmentPoints' | 'bedrockDiamonds';
+        name: string;
+        logo: string;
+        amountPerBTCPerDay: number | null;
+    }[];
```

Comment:
```suggestion
    baseApy: {
        name: string;
        value: number;
    };
    rewardApy: { .. },
    points: {
        id: string;
        name: string;
    }[];
```

Reply:
`points` make sense

I would change the rest to

```
    baseApy: number;
    rewardApy: number;
    rewardTokens: Token[]
```

reasoning is:
- baseApy: I don't know what I would put in the name
- rewardApy: DL does not give us a way to differentiate in which form the yield is coming from
- rewardTokens: DL return us rewardTokens as string[], I think we could easily transform it into `Token`s

Reply:
Sounds good to me, let's align with what Defillama gives us!